Specs/spec-main.js

```diff
@@ -136,6 +136,8 @@
 
         window.it = function(description, f, timeout, categories) {
             originalIt(description, function(done) {
+                // Clear the RequestScheduler before for each test in case requests are still active from previous tests
+                Cesium.RequestScheduler.clearForSpecs();
```

Comment:
Is this still needed here?  I would prefer to remove it unless there's a really good reason for it.